chimerapulse/core/speech/text_to_speech.py

```diff
@@ -0,0 +1,51 @@
+"""This is a helper file for core.translator.text_translator capability
+"""
+import os
+
+# Import namespaces
+import azure.cognitiveservices.speech as speech_sdk
+
+# Globals
+global speech_config
+
+
+def __authenticate_client():
+    """Authenticate service
+
+    Returns:
+        (TextTranslationClient): Authenticated client
+    """
+    global speech_config
+    
+    # Get Configuration Settings
+    translatorkey = os.getenv('SPEECH_SERVICE_KEY')
+    translatorregion = os.getenv('SPEECH_SERVICE_REGION')
+
+    # Connect to Speech Config
+    speech_config = speech_sdk.SpeechConfig(translatorkey, translatorregion)
+
+
+def synthesizeText(targetLanguage, translatedObj):
```

Comment:
let's make this accept the text to be spoken instead of the entire "translatedObj". The text to speech package shouldn't care about the object. Only the actual text you want it to synthesize